### ISSUE TITLE
9.1.0 requires "node-fetch", but not specify it as a dependency

### ISSUE THREAD

**Issue by artamonovkirill** [testcontainers/testcontainers-node#421]
**Expected Behaviour**
`npm test` works

**Actual Behaviour**
```
 FAIL  src/index.spec.js
  ● Test suite failed to run

    Cannot find module 'node-fetch' from 'node_modules/testcontainers/dist/http-wait-strategy.js'

    Require stack:
      node_modules/testcontainers/dist/http-wait-strategy.js
      node_modules/testcontainers/dist/wait.js
      node_modules/testcontainers/dist/reaper.js
      node_modules/testcontainers/dist/generic-container/generic-container.js
      node_modules/testcontainers/dist/index.js
      src/localstack.test.js
      src/index.spec.js

    > 1 | const {GenericContainer, Wait} = require('testcontainers')
        |                                  ^
      2 |
      3 | let container
      4 |

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:425:11)
      at Object.<anonymous> (node_modules/testcontainers/dist/http-wait-strategy.js:18:38)
      at Object.<anonymous> (node_modules/testcontainers/dist/wait.js:5:30)
      at Object.<anonymous> (node_modules/testcontainers/dist/reaper.js:20:16)
      at Object.<anonymous> (node_modules/testcontainers/dist/generic-container/generic-container.js:20:18)
      at Object.<anonymous> (node_modules/testcontainers/dist/index.js:4:27)
      at Object.require (src/localstack.test.js:1:34)
      at Object.require (src/index.spec.js:6:20)
```

**Testcontainer Logs**
n/a (no containers are start)

**Steps to Reproduce**
Example project: https://github.com/artamonovkirill/ninshubur/pull/221

**Environment Information**
- Operating System: macOs Ventura 13.1
- Docker Version: 20.10.20
- Node version: v19.2.0
- Testcontainers version: 9.1.0

**Comment by cristianrgreco** [comment]
@artamonovkirill Thanks! Fixed in 9.1.1

**Comment by artamonovkirill** [comment]
@cristianrgreco thanks, 9.1.1 works fine! 🙌
